includes/modules/searchandreplace/class-pb-search.php

```diff
@@ -37,7 +49,18 @@ function search_for_pattern( $search, $limit, $offset, $orderby ) {
 			if ( ! ini_get( 'safe_mode' ) ) {
 				set_time_limit( 0 );
 			}
-			return $this->find( '@' . preg_quote( $search, '@' ) . '@', $limit, $offset, $orderby );
+			if ( $this->regex ) {
+				// First test that the search and replace strings are valid regex
+				set_error_handler( array( &$this, 'regex_error' ) );
+				$valid = @preg_match( $search, '', $matches );
```

Comment:
Interesting.

On Stack Overflow, the [most popular answer for validating a regex](https://stackoverflow.com/a/12941133) uses this same procedure and recommends using `@`

It's probably better to keep it, please add `// @codingStandardsIgnoreLine` where necessary?